tests/test_process.py

```diff
@@ -0,0 +1,176 @@
+import os
+import shutil
+from collections.abc import Mapping
+
+import pytest
+from aiida import orm
+
+pytest_plugins = ["aiida.manage.tests.pytest_fixtures"]  # pylint: disable=invalid-name
```

Comment:
We do not have pyliint.
```suggestion
pytest_plugins = ["aiida.manage.tests.pytest_fixtures"]
```